packages/schematics/src/collection/ngrx/ngrx.md

```diff
@@ -1,61 +1,80 @@
-# ngrx
---------
+# Angular CLI `@nrwl/schematics:NgRx`
```

Comment:
I don't think we should have this doc here.

We have API docs deployed to `nrwl.io/nx`. This doc appears to server a similar purpose, but I don't think a lot of folks will be able to find it.

I'd remove this doc and update `nrwl.io/nx` instead.